depends/packages/xproto.mk

```diff
@@ -6,7 +6,8 @@ $(package)_file_name=$(package)-$($(package)_version).tar.bz2
 $(package)_sha256_hash=636162c1759805a5a0114a369dffdeccb8af8c859ef6e1445f26a4e6e046514f
 
 define $(package)_set_vars
-  $(package)_config_opts=--without-fop --without-xmlto --without-xsltproc --disable-specs
+  $(package)_config_opts=--without-fop --without-xmlto --without-xsltproc --disable-specs --disable-shared
```

Comment:
@div72, I am not sure why Bitcoin was building the shared libraries for this? Opinion?

Reply:
It's a header-only library. It gets preprocessed a bit, but nothing is compiled.